spec/factories/users.rb

```diff
@@ -0,0 +1,13 @@
+FactoryBot.define do
+  factory :user do
+    nickname              { Faker::Name.initials }
+    email                 { Faker::Internet.email }
+    password              { Faker::Internet.password(min_length: 6) }
```

Comment:
```suggestion
    password              { Faker::Internet.password(min_length: 6) }
```
Fakerのみですと英字のみや数字のみのpasswordが生成されてしまう可能性があります。
password {Fakerによって生成された文字列+'1a'}のように'1a'を足すことで必ず半角英数字混合になるようにしましょう。